### DISCUSSION TITLE
Is it production ready?

### DISCUSSION THREAD

**Discussion by guiaramos** [typedorm/typedorm#173] [Q&A] [answered]
I would like to know if this lib is recommended for production apps.
Thanks and great work!

**Comment by whimzyLive** [comment, marked as answer]
Hi @guiaramos  

Yes, it is production ready. we have now implemented all (_Most of_) the  document client operations in the core of this library.   

As per our current release lifecycle, any new feature added to the library will first be made available on `beta` channel for developers to review and provide feedback. (Usually around 1-4 months). Only after the feature is fully backed, it will be deployed to main channel. This allows us to confidently release new features while providing the most stability.
  
In addition, We very much try to follow semver, so there should not be breaking changes between minor feature releases.

**Reply by guiaramos** [reply]
@whimzyLive ty!
